website/docs/commands/compile.md

```diff
@@ -121,7 +133,60 @@ See the [Dependency management](../guides/dependencies.md) guide for more detail
 
 ## Scala compiler options
 
-Most [Scala compiler options](https://docs.scala-lang.org/overviews/compiler-options) can be passed as-is to `scala-cli`:
+### Passing compiler options with `-O`
+
+All [Scala compiler options](https://docs.scala-lang.org/overviews/compiler-options) can be passed to `scala-cli`
+with `-O`:
+
+```bash
+scala-cli compile Hello.scala -O -deprecation -O -Xlint:infer-any
+# [warn] ./Hello.scala:3:7: method x in class Some is deprecated (since 2.12.0): Use .value instead.
+# [warn]   opt.x
+# [warn]       ^
+```
+
+Passing a value to a compiler option requires another `-O`:
+
+```bash
+scala-cli -O -Xshow-phases -O -color -O never
+```
+
+:::note
+Scala CLI uses `bloop` by default, which at times gets in the way of getting the full compiler output.
```

Comment:
Is that only the `-explain` issue? I think @lwronski found the origin of the issue.

Reply:
Sure, makes sense. I was just worried there are more issues :sweat_smile: